stages/org.osbuild.oscap.remediation

```diff
@@ -128,15 +129,82 @@ def main(tree, options):
     if tailoring is not None:
         cmd.extend(["--tailoring-file", tailoring])
 
+    if arf_results is not None:
+        # run with command in chroot so full path ok
+        cmd.extend(["--results-arf", f"{data_dir}/{arf_results}"])
+
+    cmd.extend(["--results", f"{data_dir}/{xccdf_results}"])
+
     cmd.append(datastream)
 
-    res = subprocess.run(cmd, encoding="utf8", stdout=sys.stderr, check=False)
+    print("OpenSCAP (evaluate): ", repr(cmd))
+    # we need to inform the scanner that the system is offline (OSCAP_PROBE_ROOT="")
+    # and provide environment variables in OSCAP_CONTAINER_VARS as the probe
+    # does not have access to /proc
+    res = subprocess.run(cmd, encoding="utf8", stdout=sys.stderr, check=False,
+                         env=dict(os.environ,
+                             OSCAP_PROBE_ROOT="",
+                             OSCAP_CONTAINER_VARS="container=bwrap-osbuild"))
```

Comment:
With this variable set scanner would respect `osbuild` platform of the CaC content and won't try to enable FIPS and other stuff, which would just ruin the image if executed in way that is meant for live systems.